### PR TITLE
initial checkin

### PR DESCRIPTION
**Pull request by jeffdo**

### CODE REVIEW

File: probablefutures-py/probablefutures/output.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Input:
```
Comment (jakubwaller) [inline probablefutures-py/probablefutures/output.py]: Maybe you meant "Output"? 😁

**Review by MoustafaWehbe — CHANGES_REQUESTED** [review]
This is looking great! Thank you @jeffdo and @jakubwaller for your contribution!
Other than @jakubwaller comment on changing the Input to Output, can you also please update the README file to include a small description about the repo.

I read your comments in the README file, and we are planning this week to make some updates on the statistics query including changing `lon` to `longitude` and `lat` to `latitude` inside the input. Also we will be using numerical type instead of string for these variables.

Thank you again! Once this is resolved, we can go ahead and merge this PR.

File: probablefutures-py/probablefutures/probablefutures.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import requests
+import input
```
Comment (jakubwaller) [inline probablefutures-py/probablefutures/probablefutures.py]: After building the package and trying to use it, it didn't find the input module. I had to change it to:
`import probablefutures.input as input`

File: probablefutures-py/probablefutures/output.py (hunk 1 above)
Comment (jeffdo) [reply probablefutures-py/probablefutures/output.py]: This is fixed those these classes aren't really available for use in the API yet.  But hopefully soon

File: probablefutures-py/probablefutures/probablefutures.py (hunk 2 above)
Comment (jeffdo) [reply probablefutures-py/probablefutures/probablefutures.py]: You are probably importing at the wrong level.  The library is mean to be imported with "probablefuture-py" as the root, so right now, you have to do:  "from probablefutures.probablefutures import ProbableFutures" which is awkward.  I'll work on make all this nicer.

**Review by MoustafaWehbe — APPROVED** [review]
LGTM! Thank you @jeffdo for your contribution!

**Pull request merged**